### PR TITLE
Fix Post's author relation

### PR DESCRIPTION
**Pull request by Ilyes512**
The Post's relation `author` does not work unless you specify the `$foreignKey`. Without it, it will try to guess the foreign key based on the method name (author) so it  will try to find `author_id` instead.

The linked repo does not seem to match the tutorial: https://github.com/nuwave/lighthouse-tutorial/blame/master/app/Post.php#L9-L12

N/A

- ~[ ] Added or updated tests~
- ~[ ] Added Docs for all relevant versions~
- ~[ ] Updated CHANGELOG.md~

**Changes**

Fixed relation issue in tutorial.

**Breaking changes**

N/A

### CODE REVIEW

File: docs/tutorial/README.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ class Post extends Model
 {
     public function author(): BelongsTo
     {
-        return $this->belongsTo(User::class);
+        return $this->belongsTo(User::class, 'user_id');
```
Comment (spawnia) [inline docs/tutorial/README.md]: Let's change the migration to name the column `author_id` instead, that way we match the convention that Laravel expects.

**Comment by spawnia** [comment]
Thanks for the PR, great find. I think we should use Laravel conventions if possible, can you please adapt the PR?

File: docs/tutorial/README.md (hunk 1 above)
Comment (Ilyes512) [reply docs/tutorial/README.md]: Done

**Comment by spawnia** [comment]
Nicely done, thank you!

**Pull request merged**